roles/uchiwa/proxy/meta/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+
+dependencies:
+  - httpd
+  - uchiwa
+  - role: httpd/proxy
+    httpd_proxy_service_name: "uchiwa"
+    httpd_proxy_dest: '{{ uchiwa_proxy_dest }}'
+    httpd_proxy_user: '{{ uchiwa_proxy_user }}'
+    httpd_proxy_pass: '{{ uchiwa_proxy_pass }}'
```

Comment:
We need to also set httpd_proxy_port here because it is needed for 15-service.conf.j2.